operations/helm/charts/mimir-distributed/values.yaml

```
@@ -474,6 +494,48 @@ alertmanager:
   env: []
   extraEnvFrom: []
 
+  # -- Options to configure zone aware replication for alertmanager
+  zoneAwareReplication:
+    # -- Enable zone aware replication for alertmanager
+    enabled: false
+    # -- Maximum number of alertmanagers that can be unavailable per zone during rollout
+    maxUnavailable: 2
+    # -- topologyKey to use in pod anti-affinity. If unset, no anti-affinity rules are generated. If set, the generated anti-affinity rule makes sure that pods from different zones do not mix.
```

nit, can reject if you think this is too much explanaiton
```suggestion
    # -- topologyKey to use in pod anti-affinity. If unset, no anti-affinity rules are generated. If set, the generated anti-affinity rule makes sure that pods from different zones do not mix on nodes with the same topology key value.
```

---

I think it's fine to leave it out, those who know know